source/onboard/migrate-from-slack.rst

```diff
@@ -0,0 +1,191 @@
+9. Copy the **Bot User OAuth Token** and save it somewhere convenient.
+
+We'll now use **Bot User OAuth Token** with the ``slack-advanced-exporter`` tool to download emails and attachments. Download the latest release of ``slack-advanced-exporter`` for your OS and architecture `here <https://github.com/grundleborg/slack-advanced-exporter/releases/>`__ and extract the executable from the download.
+
+Once you have the program downloaded locally, run the commands below to fetch the emails, and then fetch file attachments. Replace ``<SLACK TOKEN>`` with the Slack token you generated earlier and ``<SLACK EXPORT FILE>`` with the `path <https://www.geeksforgeeks.org/absolute-relative-pathnames-unix/>`__ to your export file.
+
+.. code:: bash
+
+  ./slack-advanced-exporter --input-archive <SLACK EXPORT FILE> --output-archive export-with-emails.zip fetch-emails --api-token <SLACK TOKEN>
+  ./slack-advanced-exporter --input-archive export-with-emails.zip --output-archive export-with-emails-and-attachments.zip fetch-attachments --api-token <SLACK TOKEN>
+
+.. note::
+
+  - The first command collects all of the user emails and creates the file ``export-with-emails.zip``. The second command fetches attachments and creates the file ``export-with-emails-and-attachments.zip``, which we will use going forward.
+  - The second command can take a long time if you have a large number of file uploads. If it's interrupted, delete the file generated (if any), and start again.
+
+The file ``export-with-emails-and-attachments.zip`` now contains all the information necessary to be imported into Mattermost.
+
+From the slack advanced exporter repo:
+Due to archive/zip limitations, these actions cannot modify archive in place. It's preferable to fetch e-mails first to avoid copying large attachments around.
+
+.. important::
+
+  Avoid unzipping and rezipping the Slack export. Doing so can modify the directory structure of the archive which could cause issues with the import process.
+
+4. Convert Slack import to Mattermost's bulk export format
+----------------------------------------------------------
+
+Now that you have a Slack export file with emails and attachments, let's convert this information into Mattermost's bulk import format using the import preparation tool ``mmetl``. Download the latest release of ``mmetl`` for your `OS and architecture <https://github.com/mattermost/mmetl/releases/>`__. Run ``mmetl help`` to learn more about using the tool.
+
+Next, run the following command to create a Mattermost bulk import file. Replace ``<TEAM NAME>`` with the name of your team in Mattermost:
+
+.. code:: bash
+
+    ./mmetl transform slack --team <TEAM NAME> --file export-with-emails-and-attachments.zip --output mattermost_import.jsonl
+
+The tool outputs a `.jsonl <https://jsonlines.org/examples>`__ file containing all of your users, channels, and posts. It also creates a ``data`` folder that contains all of your attachments.
+
+.. important::
```

Comment:
nit: Can we use other color than green in the important blocks? they don't seem to impact that much right now. Maybe in the new docs design?

Reply:
@fmartingr - Loving your mindset here! Yes, in the new docs design, we have full control over not only the colour of callouts, but can customize the header text and icons beyond "Notes" or "Important".

I agree that "Note" blocks should be styled to appear different/distinct from "Important" and other callouts to draw attention appropriately.